dyngen/generate_dataset.R

```diff
@@ -0,0 +1,118 @@
+# Script created with the help of Robrecht Cannoodt
+library(tidyverse)
+library(dyngen)
```

Comment:
Should we add a comment about the dyngen commit or version used to run this?  Because dyngen is changing rapidly, a reader maybe confused about why they can't rerun this script to reproduce your results.

Reply:
Fixed. Added comment in the script. Should I add a comment in the README as well, perhaps?

Reply:
I think this is sufficient for now, though I wouldn't object if you want to add it to the readme as well.

I'm going to merge these major additions and you can make touchups on the master branch if you see anything else.